snapshot_create.go

```diff
@@ -0,0 +1,179 @@
+// Copyright 2012-present Oliver Eilhard. All rights reserved.
+// Use of this source code is governed by a MIT-license.
+// See http://olivere.mit-license.org/license.txt for details.
+
+package elastic
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/olivere/elastic/uritemplates"
```

Comment:
This must be `"gopkg.in/olivere/elastic.v5/uritemplates"`. The generator need some polish, I'm sorry.